packages/color-area/README.md

```diff
@@ -0,0 +1,68 @@
+## Description
+
+An `<sp-color-area>` allows users to visually select two properties of a color simultaneously. It's commonly used together with a color slider or color wheel.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/color-area?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/color-area)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-area?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-area)
+
+```
+yarn add @spectrum-web-components/color-area
+```
+
+Import the side effectful registration of `<sp-color-area>` via:
+
+```
+import '@spectrum-web-components/color-area/sp-color-area.js';
+```
+
+When looking to leverage the `ColorArea` base class as a type and/or for extension purposes, do so via:
+
+```
+import { ColorArea } from '@spectrum-web-components/color-area';
+```
+
+## Usage
```

Comment:
Can we call this `Color property` or something else that's not just "Usage"?

Also, can we add this to the `slider` and `wheel` docs, too?